### DISCUSSION TITLE
Shouldn't `visualize_feature_importance`from explanation use absolute values when applying sum ?

### DISCUSSION THREAD

**Discussion by brochier** [pyg-team/pytorch_geometric#10261] [Q&A]
HI,

I'm experimenting with the explain features of PyG these days and I looked at the feature visualisation in this module: https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/explain/explanation.py

I'm just wondering if the mask should summed like this or if we should consider their absolute values ? I have seen this kind of processing in other codes such as: https://www.tensorflow.org/tutorials/interpretability/integrated_gradients

Since the values can be negative or positive and that zero values are to be ignored, I think it would make sens to apply `torch.abs` beforehand.

What do you think ?
